csrc/dispatch_utils.h

```diff
@@ -14,3 +14,13 @@
 #define VLLM_DISPATCH_FLOATING_TYPES(TYPE, NAME, ...)             \
   AT_DISPATCH_SWITCH(                                             \
     TYPE, NAME, VLLM_DISPATCH_CASE_FLOATING_TYPES(__VA_ARGS__))
+
+#define VLLM_DISPATCH_CASE_FLOATING_BYTE_TYPES(...)         \
```

Comment:
Nit: just my personal preference
```suggestion
#define VLLM_DISPATCH_CASE_FLOATING_AND_BYTE_TYPES(...)         \
```
Similar for `VLLM_DISPATCH_FLOATING_BYTE_TYPES` below

Reply:
I also perfer to the new name which is more clear. Fixed.